### DISCUSSION TITLE
Não estou conseguindo entrar no tiktok pay

### DISCUSSION THREAD

**Discussion by mariaelaine5** [vercel/community#1578] [Help] [answered]
### Summary

Não estou conseguindo entrar no tiktok pay eu também troquei meu ponto mais até agora não recebi minha recompensa eu preciso de uma solução

**Comment by amyegan** [comment, marked as answer]
Hello @mariaelaine5. I'm sorry to hear you are having trouble. I think you are all talking about a site that is hosted by Vercel, but is not owned by Vercel. This issue has previously been discussed in https://github.com/vercel/community/discussions/1151. You should contact the site owner for help
